cwl/src/main/scala/wdl4s/cwl/Cwl.scala

```diff
@@ -42,7 +46,7 @@ case class Workflow(
 case class CommandLineTool(
                             inputs: Inputs = Coproduct[Inputs](Array.empty[CommandInputParameter]),
                             outputs: Outputs = Coproduct[Outputs](Array.empty[CommandOutputParameter]),
-                            `class`: W.`"CommandLineTool"`.T = "CommandLineTool".narrow,
+                            `class`: CommandLineTool.`class`.type = CommandLineTool.`class`,
```

Comment:
I got sporadic compiler errors telling me it found String but required String("CommandLineTool") or String("Workflow") when I tried to instantiate CommandLineTool or Workflow.

I did not always get this error when instantiating CommandLineTool or Workflow, but when I got it, I could not figure out, why or how to prevent it, except for the approach you see here. I'm suspecting the Refined library uses a macro with some odd limitations.

Either way, my reasoning is that if you want to use properly the type of a singleton, you have to create a reference to that singleton so that you can ask for its type. So this is what I did.

Thoughts?